Readme.md

```diff
@@ -50,7 +50,7 @@ export JAVA_HOME=/path/to/jdk
 ln -s /path/to/jdk ${HOME}/android-java
 ```
 
-6. Download the link https://mega.nz/file/RsMEgZqA#s0P754Ua7AqvWwamCeyrvNcyhmPjHTQQIxtqziSU4HI, uncompress it and put the folder `webrtc` in the path `app/src/main/jni/megachat/`.
+6. Download the link https://mega.nz/file/RsMEgZqA#s0P754Ua7AqvWwamCeyrvNcyhmPjHTQQIxtqziSU4HI, uncompress it and put the folder `webRTC` in the path `app/src/main/jni/megachat/`.
```

Comment:
The folder name in project is `webrtc` (all lowercase), so this change is wrong.